tier1-icons.html

```diff
@@ -714,6 +714,13 @@
             <path d="M17.83 1.55l-1.04 1.56c.14.275.213.58.21.89a2 2 0 0 1-1.88 1.99l-1.45 2.9A2 2 0 1 1 10 10v-.03L7.97 8.75c-.177.1-.37.172-.57.21l-1.45 4.36A1 1 0 0 1 5 14a1.251 1.251 0 0 1-.32-.05 1.011 1.011 0 0 1-.63-1.27L5.5 8.32A2 2 0 1 1 9 7v.03l2.03 1.22c.26-.147.552-.23.85-.24l1.45-2.9A2.007 2.007 0 0 1 15 2a.569.569 0 0 1 .13.01L16.17.45a1 1 0 0 1 1.66 1.1z"/>
           </g>
         </g>
+        <g id="visibility-conditional">
+          <path d="M5.363,8.662A2.333,2.333,0,1,1,9.333,7c0,.054-.012.1-.016.157A12.093,12.093,0,0,1,11.25,7,11.858,11.858,0,0,1,14,7.32v-1.1a.78.78,0,0,0-.778-.778h-.591a.776.776,0,0,1-.544-1.33L12.5,3.7a.784.784,0,0,0,0-1.1L11.4,1.5a.784.784,0,0,0-1.1,0l-.4.4a.757.757,0,0,1-.568.249.78.78,0,0,1-.778-.778V.778A.78.78,0,0,0,7.778,0H6.222a.78.78,0,0,0-.778.778v.591a.78.78,0,0,1-.778.778A.757.757,0,0,1,4.1,1.9l-.4-.4a.784.784,0,0,0-1.1,0L1.5,2.6a.784.784,0,0,0,0,1.1c.148.148.311.288.451.443a.715.715,0,0,1,.194.521.78.78,0,0,1-.778.778H.778A.78.78,0,0,0,0,6.222V7.778a.78.78,0,0,0,.778.778h.591a.78.78,0,0,1,.778.778.715.715,0,0,1-.194.521c-.14.156-.3.3-.451.443a.784.784,0,0,0,0,1.1l1.1,1.1.011.007A6.149,6.149,0,0,1,5.363,8.662Z"/>
+          <path d="M4.943,11.909h0c.03-.053.055-.108.088-.16C5,11.8,4.973,11.856,4.943,11.909Z"/>
+          <path d="M6.17,12.792a3.659,3.659,0,0,1,1.752-1.571A3.659,3.659,0,0,0,6.17,12.792Z"/>
+          <path d="M6.042,10.639a6.922,6.922,0,0,1,1.991-1.1A6.922,6.922,0,0,0,6.042,10.639Z"/>
+          <path d="M11.12,9C7.4,9,4.38,11.02,4.38,13.5S7.4,18,11.12,18s6.76-2.02,6.76-4.5S14.85,9,11.12,9ZM8.78,15.67c-1.43-.43-2.4-1.24-2.4-2.17s.97-1.74,2.4-2.17a3.9,3.9,0,0,0,0,4.34ZM12,14a1,1,0,1,1,1-1A1,1,0,0,1,12,14Zm1.46,1.68a3.93,3.93,0,0,0,0-4.36c1.45.43,2.42,1.25,2.42,2.18S14.91,15.25,13.46,15.68Z"/>
+        </g>
```

Comment:
I like that this slots in right next to `visibility-hide` and `visibility-show`.